Userland/Libraries/LibJS/Runtime/AbstractOperations.cpp

```diff
@@ -29,7 +28,6 @@
 #include <LibJS/Runtime/PropertyName.h>
 #include <LibJS/Runtime/ProxyObject.h>
 #include <LibJS/Runtime/Reference.h>
-#include <LibJS/Runtime/Utf16String.h>
```

Comment:
This type is used in `get_substitution` - the `replace_string` var is a `Utf16String`